docs/themes/thxvscode/layouts/partials/apiNav.html

```diff
@@ -164,104 +164,108 @@
     </ul>
     <ul class="nav" id="fluid-framework-nav">
         <li class="panel {{ if $isAnythingInFFNavActive }}active expanded{{ end }}">
-            <a class="area" href="#fluid-framework-reference" data-parent="#fluid-framework-nav" data-toggle="collapse">Fluid Framework</a>
-            <ul id="fluid-framework-reference" class="collapse {{ if $isAnythingInFFNavActive }}in{{ end }}">
-                <li class="panel {{if $isFFActive }}active{{ end }}">
-                    <a class="area{{if $isFFActive }} active expanded{{ end }} nav-page" href="{{ $FFLink }}">Complete Reference</a>
-                </li>
-                {{ range $categoryName := $.Scratch.Get "specificCategories" }}
-                    {{ $categories := index $FluidFramework $categoryName }}
-                    {{ $isCategoryActive := eq $activeCategory $categoryName }}
-                    <li class="panel {{ if $isCategoryActive }}active expanded{{ else }}collapsed{{ end }}">
-                        <a class="area" href="#{{ $categoryName }}-articles" data-toggle="collapse">{{ $categoryName }}</a>
-                        <ul id="{{ $categoryName }}-articles" class="collapse {{ if $isCategoryActive }}in{{ end }}">
-                            {{ range $subCategoryName, $APIs := $categories }}
-                                {{ $isSubCategoryActive := and (eq $activeSubCategory $subCategoryName) $isCategoryActive }}
-                                <li class="panel type-definitions {{ if $isSubCategoryActive }}active expanded{{ else }}collapsed{{ end }}">
-                                    <a class="area padding-35px" href="#{{ $categoryName }}-{{ $subCategoryName }}-articles" data-toggle="collapse">{{ $subCategoryName }}</a>
-                                    <ul id="{{ $categoryName }}-{{ $subCategoryName }}-articles" class="collapse {{ if $isSubCategoryActive }}in{{ end }}">
-                                        {{ range $api := $APIs }}
-                                            {{ $apiLink := printf "%s/%s-%s" $FFLink (lower $api) (lower $subCategoryName | singularize) }}
-                                            {{ if eq $subCategoryName "Types" }}
-                                                {{ $apiLink = printf "%s/%s-typealias" $FFLink (lower $api) }}                                               
+            {{if eq $currentApiVersion "v1" }}
```

Comment:
Can we add a comment explaining this? We have a lot of funky stuff going on in our site build that makes maintenance difficult. The more comments we have explaining the weirdness, the easier our future lives will be 🤣